src/db/models/expenses.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { sequelize } = require('../main');
+const { DataTypes } = require('sequelize');
+
+const Expenses = sequelize.define('Expense', {
```

Comment:
Consider renaming the model 'Expense' to 'Expenses' for consistency with the tableName.